atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/CacheValue.java

```diff
@@ -62,6 +62,11 @@ public boolean equals(Object obj) {
         }
     }
 
+    @Override
+    public String toString() {
+        return CacheValue.class.getSimpleName() + "{value=" + value().map(Arrays::toString) + "}";
```

Comment:
this prints differently compared to if this were an immutable wherein if the value is empty, it won't be logged at all

i think this ok but do let me know